csharp/HOLMS.Platform/HOLMS.Platform/Support/DTOBuilders/Booking/Groups/GroupBookingBuilder.cs

```diff
@@ -36,7 +36,8 @@ public class GroupBookingBuilder {
             AccountIndicator houseAccount, RateScheduleIndicator rs, GroupIndicator g,
             Guid? groupBookingMethodId,
             Guid? reservationSourceId, Guid? travelAgentId,
-            Guid? cancellationPolicy) {
+            Guid? cancellationPolicy,
```

Comment:
Hmm. "group booking cancellation policy"

I think we need to make this effective, it's just something we never got around to doing. We should also eliminate the whole travel agent id field here.

Small tweaks for later. I added a note to discuss the group booking's cancellation policy for our next meeting. 

Additionally, I believe we should eventually (perhaps after our discussion), (1) make the cancellation policy non-optional, (2) adapt Reservation.cs to honor it (either by attaching at booking time, or getting at cancellation-time), and (3) convert this field to an indicator. For now, it seems `Guid?` is the best way to handle the optional nature of this field, since I don't want to allow nullable indicators, in general.